src/Writer/ErrorLog.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Zend\Log\Writer;
+
+use Zend\Log\Formatter\Simple as SimpleFormatter;
+
+class ErrorLog extends AbstractWriter
+{
+    /**
+     * @var int
+     */
+    protected $mode;
+
+    /**
+     * @var string|null
+     */
+    protected $destination;
+
+
+    /**
+     * ErrorLog constructor.
+     * @param array|null|\Traversable $options
+     */
+    public function __construct($options = [])
+    {
+        parent::__construct($options);
+
+        $this->mode = isset($options['mode']) ? $options['mode']: 0 ;
+        $this->destination = isset($options['destination']) ? $options['destination']: null;
```

Comment:
I'd recommend validating that `$options['destination']` is valid (i.e., it needs to be a string, and you may also want to validate that it's a valid stream reference).